### ISSUE TITLE
Runtime error: gts file not able to be resolved

### ISSUE THREAD

**Issue by NullVoxPopuli** [ember-template-imports/ember-template-imports#9]
Error:
```
opcode-compiler.js:233 Uncaught (in promise) Error: Attempted to resolve `back-of-cards`, which was expected to be a component, but nothing was found.
    at resolveComponent (opcode-compiler.js:233)
    at encodeOp (opcode-compiler.js:2580)
    at pushOp (opcode-compiler.js:2504)
```

Repro: https://github.com/NullVoxPopuli/emberclear/pull/1522

**Comment by NullVoxPopuli** [comment]
when renamed to gjs, the same error I've been getting at build time occurs.

Did I miss a setup step? 🤔

**Comment by pzuraq** [comment]
`.gts` is not currently supported by this addon, we should definitely figure out support though.

**Comment by NullVoxPopuli** [comment]
This is resolved with v2.0.0+
